apps/token-manager/contracts/TokenManager.sol

```diff
@@ -62,10 +62,14 @@ contract TokenManager is ITokenController, AragonApp { // ,IForwarder makes cove
         )
         onlyInit external
     {
-
         initialized();
 
         require(_token.controller() == address(this));
+        if (!_token.transfersEnabled()) {
+            // If the token itself isn't transferable, make sure this app is also initialized to not
+            // be transferable
+            require(!_transferable);
+        }
 
```

Comment:
Is the possibility of using a transferable token as non-transferable supposed to be allowed? What's the use case for that? Isn't it a little bit confusing?
Otherwise, couldn't we just get rid of `_transferable` param on `initialize` function and just use the one from `_token`?

Reply:
What we could do is set the token's transferability in the TokenManager's `initialize()` based on `_transferable`.

Need to think about the implications of that... need to make sure the minting, assigning, etc. still works if we do this.